ivy/functional/backends/tensorflow/experimental/losses.py

```diff
@@ -1,9 +1,32 @@
 import tensorflow as tf
 from typing import Optional
+import ivy
 from ivy.func_wrapper import with_unsupported_dtypes
 from . import backend_version
 
 
+@with_unsupported_dtypes({"2.13.0 and below": "bool"}, backend_version)
+def smooth_l1_loss(
+    input: tf.Tensor,
+    target: tf.Tensor,
+    /,
+    *,
+    delta: Optional[float] = 1.0,
+    reduction: Optional[str] = "mean",
+) -> tf.Tensor:
+    abs_diff = ivy.abs(input - target)
```

Comment:
You should be using tf functions here, do not import ivy in the backend implementations